storage-playground-config.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+set -e
+
+SCRIPT_PATH="$(dirname "${BASH_SOURCE[0]}")"
+cd $SCRIPT_PATH
+
+# Basic script to be run to configure dev chain storage infrastructure.
+# Assumes one storage lead and distributor lead are already hired
+# and their identity information are stored in .env
+
```

Comment:
I'm thinking to add just a step to get the docker local host interface address which will work better as the initial node_uri to be set in the metadata.